docfx.json

```diff
@@ -651,6 +663,8 @@
                 "_csharplang/proposals/csharp-10.0/improved-definite-assignment.md": "Improved definite assignment analysis",
                 "_csharplang/proposals/csharp-10.0/async-method-builders.md": "AsyncMethodBuilder override",
 
+                "_roslyn/docs/compilers/CSharp/*.md": "C# compiler breaking changes since C# 10",
```

Comment:
Would it be better to use the exact file name for the title and description instead of a wildcard?

Reply:
Yes. I made updates for the keys that align to a single article.